ingestion_server/ingestion_server/slack.py

```diff
@@ -61,14 +61,17 @@ def _message(text: str, summary: str = None, level: Level = Level.INFO) -> None:
 
 
 def verbose(text: str, summary: str = None) -> None:
+    log.debug(text)
     _message(text, summary, level=Level.VERBOSE)
 
 
 def info(text: str, summary: str = None) -> None:
+    log.info(text)
     _message(text, summary, level=Level.INFO)
 
 
 def error(text: str, summary: str = None) -> None:
+    log.error(text)
     _message(text, summary, level=Level.ERROR)
 
 
```

Comment:
I can't add a change to the line directly but I think we should add a similar `log.info` line to the `status` method just below this!

Reply:
Doing that would duplicate the log because the status() method calls the info() method that already does a log.info().
Do you agree?

Reply:
OH, right! We should be set then 😄